### PR TITLE
Initial style guide for UI copy

### PR DESCRIPTION
**Pull request by stevegrossi**
More and more, we developers find ourselves writing interface copy explaining to users how to interact with new features. To help this writing scale, we need a style guide to keep this copy consistent and helpful. Here's my initial take on what that should be.

(Inspired in large part by this great post from Google Ventures: http://www.fastcodesign.com/3026463/from-google-ventures-5-rules-for-writing-great-interface-copy)

### CODE REVIEW

**Comment by stevegrossi** [comment]
@lessonly/developers, would love your thoughts here. For comments related to specific bullet points, please add them to the specific line so we can keep conversations threaded:

screen shot 2015-12-19 at 8 18 09 am

File: ui-text-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Tone
+
+- **Be human.** Never let technical details leak into UI copy. Instead of “The AJAX request failed with a 404,” a user-facing error message should say something like “We couldn’t save your course. Please try again.”
+- **Be helpful.** Whenever possible, if something goes wrong, provide appropriate suggestions for next actions. “Please try again” and “If this problem continues, please contact <support@lesson.ly>” aren’t awesome, but they’re better than nothing.
+- **Be cool.** Avoid exclamation points, all-caps (for emphasis, not acronyms), and bold and italic formatting. “You **MUST** upload a file in UTF8 format!” is rude.
+- **Be positive.** Prefer positive statements to negative ones, for example “Your username can only contain letters, numbers, and underscores” instead of “Your username can’t have any special characters.” It’s a small distinction, but our brains are unconsciously sensitive to them.
+- **Keep it simple.** Prefer simpler words to the ones you studied for the SAT, for example “start” over “initiate.” Not everyone using Lesson.ly had English as their first language.
```
Comment (armilam) [inline ui-text-style-guide.md]: How else are we going to augment our clientele's lexicon?

File: ui-text-style-guide.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Style
+
+- **Be specific.** Don’t say “enter” when you mean “select” from given options. Don’t say just “date” when you mean “date and time.” “Cancel” is not the same as “Back.”
+- **Be consistent.** If you say “Assigned On” in one module, don’t say “Assignment Date” in another. Always refer to the same thing with the same terminology.
+- **Say as little as necessary.** The shorter some text is, the more likely it is to be read. Don’t sacrifice clarity, but prefer headings like “Share Ownership” over “Add Additional Owners to This Lesson.”
+- **Break it up.** Avoid the TLDR effect. If you must provide extensive instructions, break them up into related paragraphs of one or two sentences each.
+- **Use correct English.** Unless you have a compelling reason otherwise, use the same English in interface text that you would in an essay. For example, end complete sentences with a period, and avoid non-words like “thru.” And sentence fragments.
```
Comment (armilam) [inline ui-text-style-guide.md]: mdmpzob
> Sentence fragment is also a sentence fragment.

**Comment by armilam** [comment]
ANOTHER *not bad* edition to the **styleguide**!

**Comment by stevegrossi** [comment]
I decided to make this a lesson: https://about.lesson.ly/preview/lesson/23096-interface-text-style-guide Let's continue any further discussion in Flowdock.

**Pull request closed without merging**